From b8f9a3a806c7c626d7ffaef5c20c4de815db4171 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Thu, 22 Mar 2018 17:14:39 +0800
Subject: [PATCH 3521/5242] MLK-17892 ARM64: dts: Add cpu cooling device to
 pmic thermal zone

commit  8583148a01da5ecc3d22b53c0ff3fe7623b3bd3e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the cpu cooling device to the PMIC thermal zone on imx8qm and
imx8qxp mek baord.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts  |   12 ++++++++++++
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    7 +++++++
 2 files changed, 19 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 8c3dd44..b9b7a0c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -1320,5 +1320,17 @@
 				type = "critical";
 			};
 		};
+		cooling-maps {
+			map0 {
+				trip = <&pmic_alert0>;
+				cooling-device =
+				<&A53_0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
+			};
+			map1 {
+				trip = <&pmic_alert0>;
+				cooling-device =
+				<&A72_0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
+			};
+		};
 	};
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 0ea06f4..a17f6fa 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -1083,5 +1083,12 @@
 				type = "critical";
 			};
 		};
+		cooling-maps {
+			map0 {
+				trip = <&pmic_alert0>;
+				cooling-device =
+				<&A35_0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
+			};
+		};
 	};
 };
-- 
1.7.9.5

